scripts/globals/spells/burn.lua

```diff
@@ -53,9 +52,8 @@ function onSpellCast(caster,target,spell)
 
                 local mbonus = caster:getMerit(tpz.merit.ELEMENTAL_DEBUFF_EFFECT)
                 DOT = DOT + mbonus/2 -- Damage
-                DOTp = DOTp + mbonus -- Stat Enfeeb
 
-                target:addStatusEffect(tpz.effect.BURN,DOT, 3, duration, tpz.effect.BURN, DOTp)
+                target:addStatusEffect(tpz.effect.BURN,DOT, 3, duration,FLAG_ERASABLE)
```

Comment:
FLAG_ERASABLE is a nil value
tpz.effectFlag.ERASABLE would have been 2, but I do not no why you'd set that argument to the value of an effect flag enum. Sorry I didn't catch that earlier. :x